scout/parse/clinvar.py

```diff
@@ -162,10 +162,17 @@ def clinvar_submission_lines(submission_objs, submission_header):
             header_key,
             header_value,
         ) in submission_header.items():  # header_keys are the same keys as in submission_objs
+
             if (
                 header_key in submission_obj
             ):  # The field is filled in for this variant/casedata object
-                csv_line.append('"' + submission_obj.get(header_key) + '"')
+                if (
+                    header_key in CLINVAR_SILENCE_IF_EXISTS.keys()
```

Comment:
CLINVAR_SILENCE_IF_EXISTS is not a very self-explanatory variable name. But I guess you wanted to have something general in case something else should be skipped in the presence of other available values? This works, but I wonder if it wound't be more simple and readable if you wrote the explicit code, perhaps with a comment? 

```
# Leave chromosome, start and stop columns empty if HGVS is available
if header_key in ['chromosome', 'start', 'stop'] and 'hgvs' in submission_obj:
```

Reply:
Yes, I have a feeling that might happen. The function was nicely general so I didn't like the idea of adding a specialised case there. The possible keys are defined in another constant around there, so I would prefer to keep these on the same level. But I can add a comment at least!